app/jinja_filters.py

```diff
@@ -350,6 +350,24 @@ def format_unordered_list(context, list_items):
     return mark_safe(context, summary_list)
 
 
+@evalcontextfunction
+def format_unordered_list_missing_items(context, possible_items, list_items):
+    summary_list = ''
+
+    if list_items and list_items[0]:
+        not_answered = [item for item in possible_items if item not in list_items[0]]
```

Comment:
If this variable is `not_answered` then it make sense to call `list_items`, `selected_items` or `selected_answered` or something like that?

Reply:
Actually I should probably call this `missing_items` to match filter name.